numbers-filtering.js

```diff
@@ -0,0 +1,75 @@
+
+var numbers = [1,2,3,4,5,6,7,8,9,10]
+
+var evens = numbers.filter(function(x) {
+   return x % 2 == 0;
+ });
+ console.log(evens);
+
+
+ //4
+ 
+ const userScore = [  
+{
+    id: 1, 
+    name: 'Olarike',
+    score: 34
+    },  
+{
+    id: 2, 
+    name: 'Taiwo', 
+    score: 12
+},  
+{
+    id: 3, 
+    name: 'Kehinde',
+     score: 27
+    } 
+    ];
+
+    //1. Where user score above 12
+    for(var i = 0; i< userScore.length;i++){
+        if (userScore[i].score > 12) {
+            console.log(userScore[i]);
+            
+        }
+    }
+   if(userScore[0].score && userScore[1].score && userScore[2].score > 12){
+    console.log();
+   }
```

Comment:
what are you trying to do here??

Reply:
Please ignore this....i was at first trying to use if statement but i later used for loop